.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,88 @@
+## 📝 Description
+
+Please provide a brief summary of the changes in this pull request. What issue(s) does this PR address? What problem(s) does it solve?
+
+Relates to #(issue-number)
+
+---
+
+## 📸 Screenshots/Screen record (if applicable)
```

Comment:
I'm not sure we need this part, or at least I can't really think of a scenario that might be immediately and directly visible in the UI. :thinking: 

Maybe a better way would be to mention public API (i.e. Postman) endpoints, which are directly effected by the changes.

Reply:
Good idea. But want we to give access to Postman to everybody? I think we must handle API changes/adding/removing some other way

Reply:
Ah, right :thinking: Maybe for the PR it's enough to mention API endpoints, e.g. `/bill/issue`, which are directly affected.

My point was more that I can't think of a use-case for the screenshots. Mentioning API-surface that changed (either Web API, or DHT events etc.) would be useful in a PR generally. I'm not sure about the format, though.

If someone just writes a sentence, that mentions that Event x and Endpoint y changed in z way, that would already be useful information for a reviewer, I guess